schema.graphql

```diff
@@ -311,3 +319,15 @@ type FpmmPoolMembership @entity {
   "Amount of liquidity tokens owned by funder"
   amount: BigInt!
 }
+type MarketProfitPerAccount @entity {
+  "account address + market address"
```

Comment:
Align this with the rest and rewrite as 
"ID as account address and market maker address"